lib/api/src/grpc/validate.rs

```diff
@@ -349,19 +336,12 @@ pub fn validate_geo_polygon_line_helper(line: &grpc::GeoLineString) -> Result<()
     Ok(())
 }
 
-pub fn validate_geo_polygon_exterior(
-    line: &Option<grpc::GeoLineString>,
-) -> Result<(), ValidationError> {
-    match line {
-        Some(l) => {
-            if l.points.is_empty() {
-                return Err(ValidationError::new("not_empty"));
-            }
-            validate_geo_polygon_line_helper(l)?;
-            Ok(())
-        }
-        _ => Err(ValidationError::new("not_empty")),
```

Comment:
the None case is checked with an extra validation in https://github.com/qdrant/qdrant/blob/6336395685187dffdcd786e1b3f6f5233541195a/lib/api/src/grpc/conversions.rs#L1475